en/companion_computer/pixhawk_rpi.md

```diff
@@ -0,0 +1,289 @@
+# Raspberry Pi Companion with Pixhawk
+
+This topic describes how to setup a Raspberry Pi ("RPi") companion companion running ROS2 on Linux Ubuntu OS, connecting to a [Pixhawk](../flight_controller/autopilot_pixhawk_standard.html) flight controller.
+
+There are different models of Pixhawk and RPi.
```

Comment:
FYI, note that I edited this to be generally Pixhawk Standard. That's because any Pixhawk standard will have the same connector standard. Many non-Pixhawk standards ones will too. But what this means is that we can immediately make it clear that this instruction will work on many other Pixhawk devices.